numpy/_core/fromnumeric.py

```diff
@@ -72,8 +89,8 @@ def _wrapreduction(obj, ufunc, method, axis, dtype, out, **kwargs):
 
     if type(obj) is not mu.ndarray:
         try:
```

Comment:
So, here, I'd tend to write it more like `_wrapfunc` - indeed, perhaps that can be used in some way?

Reply:
It was not my task to completely refactor the fromnumeric code, just to fix the failures of the tests of 86f6764